tools/calyx-ffi-macro/src/lib.rs

```diff
@@ -0,0 +1,241 @@
+#![forbid(unsafe_code)]
+
+use std::{env, path::PathBuf};
+
+use parse::{CalyxFFIMacroArgs, CalyxPortDeclaration};
+use proc_macro::TokenStream;
+use quote::{format_ident, quote};
+use syn::parse_macro_input;
+
+mod calyx;
+mod parse;
+mod util;
+
+// this is super bad, might go out of sync with interp::WidthInt
+type WidthInt = u32;
```

Comment:
Now that you added a `pub use` for `WidthInt` to `interp`, can you use that instead of redefining it here?

Reply:
Yes, I was lazy